applets/wncklet/showdesktop.c

```diff
@@ -35,6 +35,11 @@
 #include <libwnck/libwnck.h>
 #endif
 
+#ifndef HAVE_X11
+#include <gdk/gdkwayland.h>
+#define GDK_IS_X11_DISPLAY(object)        !(G_TYPE_CHECK_INSTANCE_TYPE ((object), GDK_TYPE_WAYLAND_DISPLAY))
+#endif
+
```

Comment:
and again.  Still, usages of `GDK_IS_X11_DISPLAY()` are already guarded.  Best would be to update the guards to include `GDK_WINDOWING_X11`, but that's probably not usually a problem -- if it is, just do so -- and this patch isn't gonna fix it.